ci_config.yaml

```
@@ -3,3 +3,4 @@ node_versions:
   - "19"
   - "20"
   - "lts"
+publish_version: "lts"
```

```suggestion
publish_version: "20"
```

While we're most likely going to use the LTS for publication, I think it's best we make that choice explicitly.